### PR TITLE
FirebaseCore M52 changelog updates

### PR DESCRIPTION
**Pull request by maksymmalyhin**

### CODE REVIEW

File: Firebase/Core/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-# v6.0.1 -- M48
+# v6.1.0 -- M52
+- [changed] `FIROptions.appGroupID` introduced. (#3293)
```
Comment (paulb777) [inline Firebase/Core/CHANGELOG.md]: s/changed/added/

Is there a phrase to add that would summarize what `FIROptions.appGroupID` does?

Comment (maksymmalyhin) [reply Firebase/Core/CHANGELOG.md]: Updated.

Comment (paulb777) [reply Firebase/Core/CHANGELOG.md]: Did you see the question I asked?

Comment (maksymmalyhin) [reply Firebase/Core/CHANGELOG.md]: Ah, sorry, I missed it.

File: Firebase/Core/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
-# v6.0.1 -- M48
+# v6.1.0 -- M52
+- [added] `FIROptions.appGroupID` property added to configure the App Group identifier required to share 
+  data between the application and the application extensions. (#3293)
```
Comment (maksymmalyhin) [inline Firebase/Core/CHANGELOG.md]: Does it look better?

**Pull request merged**